prototypes/cel-portable/fields.go

```
@@ -0,0 +1,368 @@
+				common = common || all
+			}
+			if !common {
+				return map[string]any{"status": "model-error"}
+			}
+		}
+	}
+	// Match input declaration order for deterministic probe diagnostics.
+	keys := inputOrder
+	if keys == nil {
+		dec := json.NewDecoder(strings.NewReader(string(raw)))
+		dec.Token()
```

golangci-lint (2.13.2)

[error] Error return value of `dec.Token` is not checked

(errcheck)